lib/evented/index.js

```diff
@@ -19,7 +19,37 @@ var ArrayProto = Array.prototype;
 var nativeIndexOf = ArrayProto.indexOf;
 var slice = ArrayProto.slice;
 
-function bind(event, fn) {
+/**
+ * Initialize a custom error handler or default to the BV Loader error reporting
+ * regardless of application. This will automatically be called at the first
+ * invocation of the ".on" method, and can be 
+ * event
+ * @param {Function} fn - the custom callback to be invoked whenever an error is
+ *  detected as the result of a trigger event
+ */
+function setErrorHandler(fn) {
+  this.errorHandler = function(e) {
+    global.console.error(e);
+
+    if (fn) {
+      fn.apply(this, slice.call(arguments))
+    } else {
+      if (global && global.BV && global.BV._private) {
+        global.BV._private.errorReporter()
+          .then(function(reportError) {
+            reportError(
+              new Error(e),
+              {
+                extra: arguments
+              }
+            )
+          });
+      }
+    }
+  }
+}
+
+function bind(event, fn, errorHandler) {
```

Comment:
I don't understand the point of adding an errorHandler argument to this method.  If the caller wants to catch errors from its listener, then it should just wrap its listener/code in its own try-catch.

I plan to remove this argument.